src/ReefGuideAPI.jl

```diff
@@ -136,8 +136,16 @@ function setup_regional_data(config::Dict)
         @debug "Using previously generated regional data store."
     elseif isfile(reg_cache_fn)
         @debug "Loading regional data cache from disk"
-        @eval const REGIONAL_DATA = deserialize($(reg_cache_fn))
-    else
+        # updates to packages like DiskArrays can break deserialization
+        try
+            @eval const REGIONAL_DATA = deserialize($(reg_cache_fn))
+        catch err
+            @warn "Failed to deserialize $(reg_cache_fn)" err
```

Comment:
```suggestion
            @warn "Failed to deserialize $(reg_cache_fn) with error:" err.msg
```

Reply:
`err.msg` doesn't work
> ERROR: type TypeError has no field msg

`err` works fine, it prints

```
┌ Warning: Failed to deserialize .cache\regional\regional_cache.dat with error:
│   err = TypeError: in Type{...} expression, expected UnionAll, got Type{DiskArrays.Chunked{:Core}}
```